assets/js/modules/analytics-4/components/common/PropertySelect.js

```diff
@@ -114,7 +114,10 @@ export default function PropertySelect( {
 		return <ProgressBar height={ 100 } small />;
 	}
 
-	const isValidSelection = isValidPropertySelection( propertyID );
+	const isValidSelection =
+		propertyID === undefined
+			? isValidPropertySelection( propertyID )
+			: true;
```

Comment:
Shouldn't the logic here be reversed? I thought the issue was the red border appeared even when no property ID selection was made, so we'd want it to be valid when `undefined` and _check_ if it's valid when it's defined, eg:

```suggestion
	const isValidSelection =
		propertyID === undefined
			? true
			: isValidPropertySelection( propertyID );
```